### ISSUE TITLE
questions about training details

### ISSUE THREAD

**Issue by FrankCast1e** [borisdayma/dalle-mini#83]
Hi!
     I think this is a great project! To understand the whole process, I have some questions about the details of training. 
1. Since you use "gradient accumulation", what does the x coordinate in this picture represent？The number of update or the number of training steps? 
![image](https://user-images.githubusercontent.com/47963924/129159722-edecd8bf-6c4a-462e-87a6-be0ae4a49d07.png)
2. What is the final loss or ppl of training dalle-mini?

**Comment by borisdayma** [comment]
Hi,

1. the step on the x-axis does not consider gradient accumulation, so this is each forward pass
2. our final loss gets to about 5.1

**Comment by FrankCast1e** [comment]
Got it, thank you very much!
